File: 2.0/java/me/getCallerIds.java
import com.callfire.api.client.CallfireClient;
import com.callfire.api.client.api.account.model.CallerId;

import java.util.List;

class ApiClientSample {
    public static void main(String[] args) {
        CallfireClient callfireClient = new CallfireClient("api_login", "api_password");
        List<CallerId> callerIds = callfireClient.meApi().getCallerIds();
    }
}

File: 2.0/csharp/me/getCallerIds.cs
using System.Collections.Generic;
using CallfireApiClient;
using CallfireApiClient.Api.Account.Model;

public class ApiClientSample
{
    public static void Main(string[] args)
    {
        var client = new CallfireClient("api_login", "api_password");
        IList<CallerId> callerIds = client.MeApi.GetCallerIds();
    }
}
